src/main/java/com/google/flourbot/macro_classes/Macro.java

```diff
@@ -0,0 +1,37 @@
+package com.google.flourbot.macro_classes;
+
+public class Macro {
+    public String creatorId;
+    public String macroName;
+    public String docId;
+    
+    public Macro (String creatorId, String macroName, String docId) {
+        this.creatorId = creatorId;
+        this.macroName = macroName;
+        this.docId = docId;
+    }
+
+    public void setCreatorId (String creatorId) {
+        this.creatorId = creatorId;
+    }
+
+    public void setMacroName (String macroName) {
+        this.macroName = macroName;
+    }
+
+    public void setDocId (String docId) {
+        this.docId = docId;
+    }
+
+    public String creatorId () {
+        return creatorId;
+    }
+
+    public String getMacroName () {
+        return macroName;
+    }
+
+    public String getDocId () {
+        return docId;
+    }
+}
```

Comment:
@SydneyCaulfeild I like this class-- thinking ahead!! Macro Object will be useful later on

One question - I'm unsure if we need setters, since initializing them with the constructor should be sufficient. Do we anticipate needing to change them? I remember Marek talking about avoiding having public setter methods since we don't want the wrong things to be able to modify the object (in a code review for week 5). Maybe ask him to double check?

Reply:
@lipczak-g

Reply:
Thanks for the advice, and yes Marek said we don't need setters since it should be immutable.